src/logger.js

```diff
@@ -0,0 +1,48 @@
+import { saveAs } from "file-saver";
+
+function handleErrorMessage(msg, url, line) {
+  logger.log("window.onerror - ERROR", msg, url, `line: ${line}`);
+}
+
+const logger = {
+  log: function () {
+    if (!window.logsEnabled) return;
```

Comment:
I feel like returning nothing/`undefined` once like here by doing `return;` , 
means you have to check arguments for being undefined everywhere forever.
To me that's worse than everything that might be a little more convenient by just doing `return;`
https://gist.github.com/delbetu/3fed29d1f56088938b019bc3b99bba72#the-null-defense
`